module-4/backend/week_three.md

```diff
@@ -4,8 +4,66 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. At a high level, what is Node?
+
+Node is a backend language written in JavaScript usually used for servers.
+
+2. What is Express? Why is Express similar to in the Ruby world?
+
+Express is a framework for Node projects
+
+3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+
+```javascript
+const express = require('express');
+const router  = express.Router();
+
+router.get('[PATH]', [CALLBACK]);
```

Comment:
Remember that the `[CALLBACK]` is deterministic: `(request, response, next)` with `next` as optional.